### PR TITLE
Create universal fern tree

### PR DESCRIPTION
**Pull request by ydzhou**
Create an universal fern tree for each left and right drawer

### CODE REVIEW

File: doc/fern.txt [hunk 1]
```diff
@@ -421,6 +421,15 @@ VARIABLE						*fern-variable*
...
 	Default: 0
 
+*g:fern#disable_drawer_universal*
+	Set 1 to disable the universal fern trees for left and right position
+        respectively.
+
+        If enabled, there is only one left or right drawer across the all the windows
+        for a given VIM instance.
+
+        Default: 1
```
Comment (lambdalisue) [inline doc/fern.txt:431]: This new behavior should be "opt-in" but the default value of variable should be 0.

So maybe `g:fern#disable_drawer_tabpage_isolation` or whatever is preferable.

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: doc/fern.txt (hunk 1 above)
Comment (ydzhou) [reply doc/fern.txt]: Done

File: autoload/fern/internal/viewer/auto_duplication.vim [hunk 2]
```diff
@@ -1,5 +1,5 @@
 function! fern#internal#viewer#auto_duplication#init() abort
-  if g:fern#disable_viewer_auto_duplication || !g:fern#disable_drawer_universal
+  if g:fern#disable_viewer_auto_duplication || g:fern#disable_drawer_tabpage_isolation
```
Comment (lambdalisue) [inline autoload/fern/internal/viewer/auto_duplication.vim:2]: You need to check if the viewer is drawer or not. See the original PR

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: autoload/fern/internal/viewer/auto_duplication.vim (hunk 2 above)
Comment (ydzhou) [reply autoload/fern/internal/viewer/auto_duplication.vim]: Done

**Review by lambdalisue — APPROVED** [review]

**Comment by lambdalisue** [comment]
Shall I merge it? or wait the discussion on #402 ? @ydzhou

**Comment by harderq** [comment]
> Shall I merge it? or wait the discussion on #402 ? @ydzhou

Yeah. Please merge it.

**Comment by ydzhou** [comment]
> Shall I merge it? or wait the discussion on #402 ? @ydzhou

Yeah, please merge it.

**Pull request merged**